tests/runtests.py

```diff
@@ -518,6 +518,21 @@ def test_cmd_highlight(self):
         self.assertIn("\x1b[35m44444444\x1b[0m", res)
         return
 
+    def test_cmd_aliases(self):
+        # test add functionality
+        add_res = gdb_start_silent_cmd("aliases add a b")
+        self.assertNoException(add_res)
+        # test list functionality
+        list_res = gdb_start_silent_cmd("aliases list")
+        self.assertNoException(list_res)
+        self.assertIn("a                               →  b", list_res)
```

Comment:
change this to a few checks: grab the line with the arrow, maybe, and then check for a and b, ideally use more unique words than a and b. This way you don't have to check for a specific number of spaces, which is fragile.